File: BabyCareX/app/src/main/java/com/konyol/babycarex/data/model/Barang.kt
package com.konyol.babycarex.data.model

data class Barang(
    val id: Int? = null,
    val id_kategori: Int? = null,
    val id_peminjam: Int? = null,
    val namabarang: String,
    val merk: String,
    val deskripsi: String,
    val harga: Int,
    val gambar: String? = null,
    val status: String
)

File: api-BabyCare/app/Models/Barang.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Barang extends Model
{
    use HasFactory;

    protected $fillable = [
        'id_kategori',
        'id_peminjam',
        'namabarang',
        'deskripsi',
        'merk',
        'harga',
        'gambar',
        'status'
    ];
}